src/ui.tsx

```diff
@@ -603,6 +607,20 @@ function PlotButtons(
   </div>;
 }
 
+function ResetToDefaultsButton() {
+  const dispatch = useContext(DispatchContext);
+  const onClick = () => {
+    // Utilize the default state to clear all user settings that have been saved
```

Comment:
```suggestion
    // Clear all user settings that have been saved and reset to the defaults
```

Reply:
That was a dumb comment on my part. Thanks for cleaning it up!